components/modal/style/confirmCmp.tsx

```diff
@@ -63,6 +63,9 @@ const genModalConfirmStyle: GenerateStyle<ModalToken> = (token) => {
         flexDirection: 'column',
         flex: 'auto',
         rowGap: token.marginXS,
+      },
+
+      [`${confirmComponentCls}-paragraph:nth-last-child(1)`]: {
```

Comment:
Any selectors deps on order is not reliable.

Reply:
@afc163 But, body can't fill in Modal.
So, we should change layout of modal.
(pixcel base to flex)
Are you agree to change layout of modal?
All I can think of is a flex layout solution.